frontend/src/app/features/work-packages/routing/wp-view-page/wp-view-page.component.ts

```diff
@@ -108,6 +108,29 @@ export class WorkPackageViewPageComponent extends PartitionedQuerySpacePageCompo
     this.text.button_settings = this.I18n.t('js.button_settings');
   }
 
+  breadcrumbItems() {
+    const items = [
+      {
+        href: this.pathHelperService.projectPath(this.currentProject.identifier as string),
+        text: this.currentProject.name,
+      },
+      {
+        href: this.pathHelperService.projectWorkPackagesPath(this.currentProject.identifier as string),
+        text: this.I18n.t('js.work_packages.work_packages_label'),
+      },
+      this.selectedTitle ?? '',
+    ];
+
+    if (this.isGantt) {
+      items[1] = {
+        href: this.pathHelperService.projectGanttChartsPath(this.currentProject.identifier as string),
+        text: this.I18n.t('js.work_packages.gantt_charts_label'),
+      };
```

Comment:
This is very hacky. Please assign the value conditionally instead overwriting it based on the index in the array.

```suggestion
    const items = [
      {
        href: this.pathHelperService.projectPath(this.currentProject.identifier as string),
        text: this.currentProject.name,
      },
      this.breadcrumbModuleEntry();
      this.selectedTitle ?? '',
    ];
    
    ... 
    
    
    breadcrumbModuleEntry():{href: string, text:string} {
      if (this.isGantt) {
       return ... 
      } else {
      return ...
    }
```